app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/Edit.tsx

```diff
@@ -147,7 +147,9 @@ export const Edit = ({ formId }: { formId: string }) => {
             if (element) {
               const questionNumber = getQuestionNumber(element, elementTypes);
               const item = { ...element, index, questionNumber };
-              return <ElementPanel elements={sortedElements} item={item} key={item.id} />;
+              return (
+                <ElementPanel elements={sortedElements} item={item} key={item.id} formId={formId} />
```

Comment:
Hopefully in the future we can use a react context for situations like this (lots in the app). Just a future refactoring note when/if we get around to refactoring the app state management and definitely something for outside of this PR.